PluginGenerator/RulesPluginBuilder.cs

```diff
@@ -89,7 +89,11 @@ protected override void ApplyConfiguration(string baseWorkingDirectory)
         {
             this.ValidateRulesConfiguration();
 
-            string uniqueId = Guid.NewGuid().ToString();
+            string packageName = this.FindPackageName(); ;
+            Debug.Assert(!string.IsNullOrWhiteSpace(packageName), "Package name should be set");
+
+            string resourceRoot = packageName.Replace(".", "/") + "/resources/";
+            string absoluteResourceRoot = "/" + resourceRoot;
```

Comment:
The rule and sqale xml files used to made unique using guids, which was ugly and harder to work with. They are now qualified using the package namespace.